### ISSUE TITLE
[Baker] Links & Accessbility

### ISSUE THREAD

**Issue by lesa-mcpeak** [washingtonstateuniversity/wsuwp-plugin-spine-themes#13]
https://stage.web.wsu.edu/lesa-land/
Shouldn't the underline used to distinguish the text as a link be a 4.5:1 color contrast ratio?
The new crimson is only a 1.08:1 color contrast ration to the text color (#4e4e4e). This means you need to underline the link but the underline used (#f3f3f3) fails for contrast to the background. Text and link text to background needs to be 4.5:1 and text to link text needs to be 3:1. The underline used to distnguish the link text is only 1.11:1 to the background.

**Comment by danbleile** [comment]
I'll bump the underline to #767676 that should give a 4:5:1

**Comment by danbleile** [comment]
Looks like this was actually on Adams theme.
